mixer/adapter/stackdriver/config/config.proto

```diff
@@ -153,7 +153,12 @@ message Params {
         // This field will be ignored for non-distribution metric kinds.
         BucketsDefinition buckets = 3;
 
+        // Stackdriver metric type name. If this is not provided, a
+        // concantenation of custom metric prefix (custom.googleapis.com/) and
+        // Istio metric name will be used.
+        string metric_type = 4;
```

Comment:
nit: can we add an example of a full, istio-related value perhaps?

Reply:
Done.